modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -233,6 +231,58 @@ function FieldBase({
 			type === 'radio');
 	const showPopover = fieldName === 'inputMaskFormat';
 
+	useLayoutEffect(() => {
+		console.log(type);
+		focusRequiredElement();
+	});
+
+	function focusRequiredElement() {
+		let errorInput = null;
```

Comment:
@joaovictoralvesmartins, do you need to set it to `null` or assign no value is enough? If we don't need to assign `null`, let's declare it with no value. :wink:

Reply:
I'll remove the `null` assignment.